build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -286,15 +317,16 @@ class SeleniumWebDriverController {
     ]);
     const resultWidth = Number(updatedHtmlElementSizes[0]);
     const resultHeight = Number(updatedHtmlElementSizes[1]);
-    // TODO(sparhami) These are throwing errors, but are not causing the test
-    // to fail immediately,.Figure out why, we want the test to fail here
-    // instead of continuing.
-    expect(resultWidth).to.equal(
-        width,
-        'Failed to resize the window to the requested width.');
-    expect(resultHeight).to.equal(
-        height,
-        'Failed to resize the window to the requested height.');
+
+    if (resultWidth != width || resultHeight != height) {
+      this.logError_(new Error(
```

Comment:
Should have a better name than `logError_`, it implies that the error is only logged and not thrown. You should call it something like `logAndThrowError_`.